ssssort.h

```diff
@@ -246,9 +245,8 @@ struct Classifier {
      * classify_unroll before to fill the bktout and bktsize arrays.
      */
     template <std::size_t U>
-    inline void
-    distribute(InputIterator in_begin, InputIterator in_end,
-               OutputIterator out_begin)
+    void distribute(InputIterator in_begin, InputIterator in_end,
```

Comment:
why did you remove the `inline` specifier? (I know it's mostly ignored by most compilers)